aging_due_report/wizard/wizard.py

```diff
@@ -585,9 +511,61 @@ def compute_lines(self, partner_ids):
                 aawp_ids[partner_id, currency_id] = aawp_id.id
             line['aawp_id'] = aawp_ids[partner_id, currency_id]
             line['aaw_id'] = wzd_brw.id
+            line['due_days'] = self._get_due_days(today, line)
             aawd_obj.create(line)
+
+        # Only to trigger _get_amount
+        wzd_brw._get_amount()
         return True
 
+    def _get_amount(self):
```

Comment:
@moylop260 I have taken away computed fields and consolidated `_get_amount` method into only one.

Though I would like to have implemented when creating records, I have not being able to do it without 
modifying the _get_amount_span().

Could you please further advise on this PR and give us your +1 if apply.

Regards.